WooCommerce/WooCommerceTests/ViewRelated/Blaze/BlazeCampaignCreationCoordinatorTests.swift

```diff
@@ -0,0 +1,148 @@
+import XCTest
+import Yosemite
+import protocol Storage.StorageManagerType
+import protocol Storage.StorageType
+@testable import WooCommerce
+
+final class BlazeCampaignCreationCoordinatorTests: XCTestCase {
+    /// Mock Storage: InMemory
+    private var storageManager: StorageManagerType!
+
+    /// View storage for tests
+    private var storage: StorageType {
+        storageManager.viewStorage
+    }
+
+    override func setUp() {
+        super.setUp()
+        storageManager = MockStorageManager()
+     }
+
+     override func tearDown() {
+         super.tearDown()
+     }
+
+    func test_given_disabled_i3_featureflag_when_no_product_id_supplied_then_navigate_to_webview() {
```

Comment:
Nit: The presence of the product ID doesn't affect the expected result, so we should remove it from the test name. Also, it's more straightforward to mention the expected result first before the condition:

```suggestion
    func test_webview_is_presented_when_blazei3NativeCampaignCreation_is_disabled() {
    }
```

Reply:
> Also, it's more straightforward to mention the expected result first before the condition:

I was following the example on [this documentation:](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/naming-conventions.md#test-methods)


> `func test_evolvePokemon_when_passed_a_Pikachu_then_it_returns_Raichu()`